web/stories/Molecules/Button/PrimaryRegular.stories.tsx

```diff
@@ -24,25 +25,16 @@ PrimaryColor.args = {
   children: "button",
 };
 
-export const SecondaryColor = Template.bind({});
+export const IconRight = Template.bind({});
 
-SecondaryColor.args = {
-  isColor: "secondary",
-  children: "button",
-};
-
-export const AccentCoolLightestColor = Template.bind({});
-
-AccentCoolLightestColor.args = {
-  isColor: "accent-cool-lightest",
-  children: "button",
-};
-
-export const AccentCoolDarkerColor = Template.bind({});
-
-AccentCoolDarkerColor.args = {
-  isColor: "accent-cool-darker",
-  children: "button",
+IconRight.args = {
+  isColor: "primary",
+  children: (
+    <>
+      <>Button</>
+      <Icon className="usa-icon--size-3 margin-left-05">arrow_drop_down</Icon>
+    </>
```

Comment:
I notice we have this pattern of icons with slightly different styles a bunch which should probably be consolidates in some fashion but for the moment I'm going to leave it as there is just a bit to much diversity to tackle it in this ticket

Reply:
As I mentioned earlier, the children elements of a design button can be anything, including graphics or text. I don't believe we need to address this, but I'd like to discuss it further after you've had a chance to read my thoughts.

Reply:
So the concern here is about inconsistency over the size of icons and the padding between icons and text which is all left to the parent and is already inconsistency between various places in the code base which is bad. I was thinking maybe we could obfuscate it somehow to being more determined via props but maybe not and maybe its not worth it or a good pattern. Just a thought

Reply:
I'll for sure leave it for now at least